* setting shrpdir to where the lib is really going to end up, make the
  Makefile use an install_name that also refers to that place, instead
  of perl's default.

--- Makefile.SH
+++ Makefile.SH
@@ -213,7 +213,7 @@
 # installman commandline.
 bin = $installbin
 scriptdir = $scriptdir
-shrpdir = $archlibexp/CORE
+shrpdir = $prefix/lib
 privlib = $installprivlib
 man1dir = $man1dir
 man1ext = $man1ext
